Timeline.h

```diff
@@ -15,11 +15,15 @@ class Timeline {
 public:
     Timeline();
 
+	void wipe();
+
     void begin(unsigned long startTime);
     bool isActive();
     void update(unsigned long time);
 
     TweenDuino::Tween* addTo(float& target, float to,  unsigned long duration);
+    TweenDuino::Tween* addTo(float& target, float to,  unsigned long duration, TweenDuino::Tween::Ease e, TweenDuino::Tween::EaseType type);
+    
```

Comment:
Eeep. I can't believe I missed this.  Thank you!

Reply:
@JasXSL found that one. Surprised the compiler hadn't complained.